### DISCUSSION TITLE
What is the difference between TheXtech and 1.3.0.1?

### DISCUSSION THREAD

**Discussion by M0REKZ** [Wohlstand/TheXTech#522] [Q&A]
Since ThexTech is based on SMBX 1.3, What would be all the differences on the engine and assets with SMBX 1.3.0.1?

(this doubt is killing me)

**Comment by Wohlstand** [comment]
TheXTech's "Super Mario Bros. X" assets are already different from the original SMBX 1.3 assets and partially different from SMBX 1.3.0.1. Some updates are exclusive or taken from X2 assets:

- Intro screen remains the same as SMBX 1.3
- Credits scene music remains the same as SMBX 1.3
- All music and resources have been replaced with pure-quality equivalents
- A pile of brand-new sounds has been added (NPC frozen, ice break, swooper flap, fixed wart death, own Larry death, own Ludwig death, P-Switch timeout signal)
- Castle BGOs got been updated
- Sizable blocks with rounded corners at the bottom were partially taken from X2 assets
- Updated switch blocks and switch barriers were taken from X2 assets
- Fixed beech koopas were taken from X2 assets
- Cave blocks were taken from SMBX 1.3.0.1 assets
- Wooden blocks were taken from SMBX 1.3.0.1 assets
- SMW castle fixed blocks were taken from SMBX 1.3.0.1 assets
- SMW Bonus room blocks with a tuned colour were taken from SMBX 1.3.0.1 assets
- Player switches and barriers were taken from X2 assets
- Few NPCs were replaced compared to their originals
- Playable Toad sprites have been updated
- There is a pile of fallback masks provided to ensure old episodes won't be broken because they are recoloured that expects old sprites
- Splash logo has been updated (the "TheXTech" sub-title has been added)
- The URL at the bottom of the title screen has been updated with "wohlsoft.ru"
- The loading screen splash has been updated to put the "Nintendo Switch"

**Comment by LoveBodhi** [comment]
However block-79 cames from **SMBX 1.3**, instead of SMBX2.

**Comment by M0REKZ** [comment]
Oh ok, then, as i can see, TheXTech have the assets from 1.3 but with better quality, and some taken from 1.3.0.1 and 2.0

Thanks for clarifying :D

**Comment by Yave-Yu** [comment]
1.3.0.1 is basically 1.3, with horrible running efficiency, especially running on Windows 10. TheXTech did great job 🦊

**Reply by vitzie629** [reply]
TheXTech can run on Windows, Max, & Linux, 1.3.0.1 on the other hand just runs on Windows since it’s basically just like 1.3!

**Comment by Wohlstand** [comment]
I'll move this into Discussions.
